code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -711,8 +711,8 @@
 		var/mob/living/carbon/human/H = M
 		var/datum/species/ethereal/E = H.dna?.species
 		E.adjust_charge(5*REM)
-	else if(prob(25)) //scp13 optimization
-		M.electrocute_act(rand(10,15), "Liquid Electricity in their body", 1) //lmao at the newbs who eat energy bars
+	else if(prob(3)) //scp13 optimization
```

Comment:
scp13?

Reply:
No idea. I assumed someone knew what it meant, and declined to change it. Realistically, what that line is is the probability, in percentage, that it will cause the effect. The reason I reduced it to 3 was to nerf it into the ground to prevent ethereal blood from being used as a weapon effectively when it can be made easily.

Reply:
To clarify what I changed actually does in that aspect.

Before my change: 5u= almost certain hardstun for whole duration, severe burns. 10u has a decent change of crit.

After my change: 5u= unlikely to stun you at all, but if it does: minor burns. Will not stun you for whole duration unless you are stupid unlucky. Requires at least 10u to have a chance to stun you once with a reasonable probability at a random point over the course of the entire metabolism.

As best as I can determine with my tests, increasing the amount of it injected doesn't stack the probability of the shock, it only gives it more chances due to being in the system for a longer period of time.